backend/spanish.txt

```diff
@@ -321,10 +321,10 @@ current_screen_resolution_is=La resolución actual es:
 screen_resolution=Resolución de pantalla
 width=Anchura
 height=altura
-screen_layout_save_success=Distribución de la pantalla guardada con éxito
+screen_layout_save_success=Layout de la  pantalla guardada con éxito
```

Comment:
Please remove the double space between "la" and "pentalla". Other than that, it is OK, I will merge it afterwards.